devices/tuya.js

```diff
@@ -1013,11 +1013,15 @@ module.exports = [
             .withDescription('Recover state after power outage')],
     },
     {
-        fingerprint: [{modelID: 'TS011F', manufacturerName: '_TZ3000_dpo1ysak'},
-            {modelID: 'TS011F', manufacturerName: '_TZ3000_cphmq0q7', applicationVersion: 69}],
+        fingerprint: [
+            {modelID: 'TS011F', manufacturerName: '_TZ3000_dpo1ysak'},
+            {modelID: 'TS011F', manufacturerName: '_TZ3000_cphmq0q7', applicationVersion: 69},
+            {modelID: 'TS011F', manufacturerName: '_TZ3000_5f43h46b'},
```

Comment:
Could you confirm that power measurements are not working when adding it to https://github.com/Koenkk/zigbee-herdsman-converters/pull/3363/files#diff-7ae23d3c13c2d7f7e37106590684c791aba8093f8be813b7c74a4f1484017c3bR980 ?

Reply:
Yep, it only works with this one =/